crates/cdk/src/mint/melt.rs

```diff
@@ -48,6 +48,32 @@ impl Mint {
         }
     }
 
+    fn check_amount_less_invoice(
```

Comment:
i don't think this can be in side `check_melt_request_acceptable` because  inside `check_amount_less_invoice` we are checking to the optional amount field value is equal to the amount inside the invoice request (payment_quote)

and right now check_melt_request_acceptable is handle before  `ln.get_payment_quote()`